test/e2e/extensions/high_availability_test.go

```diff
@@ -41,18 +42,35 @@ func TestGameServerHealthyAfterDeletingPodWhileOneExtensionsDown(t *testing.T) {
 
 	list, err := getAgoneseExtensionsPods(ctx)
 	logger.Infof("Length of pod list is %v", len(list.Items))
-	assert.NoError(t, err, "Could not get list of Extension pods")
+	for i := range list.Items {
+		logger.Infof("Name of extensions pod %v: %v", i, list.Items[i].ObjectMeta.Name)
+		logger.Infof("Host IP %v", list.Items[i].Status.HostIP)
+		logger.Infof("Pod IPs %v", list.Items[i].Status.PodIPs)
+	}
+	require.NoError(t, err, "Could not get list of Extension pods")
 	assert.Greater(t, len(list.Items), 1, "Cluster has no Extensions pod or has only 1 extensions pod")
 
-	logger.Info("Removing one of the Extensions Pods")
+	logger.Infof("Removing one of the Extensions Pods: %v", list.Items[1].ObjectMeta.Name)
 	deleteAgonesExtensionsPods(ctx, t)
+	require.Eventually(t, func() bool {
+		extList, err := getAgoneseExtensionsPods(ctx)
+		if err != nil {
+			return true
+		}
 
-	assert.NoError(t, waitForAgonesExtensionsRunning(ctx))
+		if len(extList.Items) == 2 {
+			return true
+		}
+
+		return false
+	}, 30*time.Second, 20*time.Millisecond)
```

Comment:
```suggestion
	}, 5 * time.Minute, time.Second)
```

We don't need to hammer the K8s API and slow down other e2e tests. Once a second should be sufficient.